apps/web/public/sw.js

```
@@ -87,6 +87,54 @@ self.addEventListener("push", (event) => {
+          headers: { "Content-Type": "application/json" },
+          body: JSON.stringify({ endpoint, keys }),
+        })
+        if (!res.ok) throw new Error("Server returned " + res.status)
+      } catch (err) {
+        // SW-side re-subscribe failed — fall through to notify clients
+        console.warn("SW pushsubscriptionchange: re-subscribe failed", err)
+      }
+
+      // Also notify any open tabs so the client-side hook can re-subscribe
+      const clients = await self.clients.matchAll({ type: "window" })
+      clients.forEach((client) => client.postMessage({ type: "PUSH_SUBSCRIPTION_CHANGED" }))
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)